hastexo-guacamole-client/routing.py

```diff
@@ -0,0 +1,22 @@
+import os
+import django
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'hastexo-guacamole-client.settings') # noqa E501
+django.setup()
+
+from channels.auth import AuthMiddlewareStack  # noqa E402
```

Comment:
Silly question, is there a specific reason that all these `import` statements come _after_ `django.setup()`?

Reply:
Yes, this was a solution to an error that I don't remember at the moment but I will double check.

Reply:
I believe this fixed the following error, when running the `hastexo_guacamole_client`:
`django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.`

Reply:
Ack, thanks!